src/main/java/io/vertx/ext/eventbus/bridge/tcp/BridgeEventType.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2014 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.ext.eventbus.bridge.tcp;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+/**
+ * Bridge Event Types.
+ *
+ * @author <a href="http://tfox.org">Tim Fox</a>
+ */
+@VertxGen
+public enum BridgeEventType {
```

Comment:
I think we should propose moving this class to `vertx-bridge-common` so all bridges can use it.